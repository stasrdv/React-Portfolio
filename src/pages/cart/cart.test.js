import React from "react";
import { shallow } from "enzyme";
import { Cart } from "./Cart";

import jsonData from "../../shared/data.json";

const items = jsonData;
describe("Cart component", () => {
  test("should shallow correctly", () => {
    expect(shallow(<Cart items={items} />)).toMatchSnapshot();
  });

  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Cart items={items} debug />);
    expect(component).toMatchSnapshot();
  });
});
