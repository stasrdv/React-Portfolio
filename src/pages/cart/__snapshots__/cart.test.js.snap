// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart component should render correctly in "debug" mode 1`] = `
<div
  className="jumobtron board"
>
  <div
    className="row justify-content-center"
  >
    <Card
      deleteItem={[Function]}
      description="React native todolist implementation"
      id={4}
      img="https://raw.githubusercontent.com/lucasbento/react-native-actions/master/common/media/logo.png"
      isCartMode={true}
      key="4"
      title="React-Native-ToDoList"
      url="https://github.com/stasrdv/React-Native-ToDoList"
    />
    <Card
      deleteItem={[Function]}
      description="WebSockets simple chat example (Socket.io ,ExpressJs,VanillaJS)"
      id={1}
      img="https://cdn-images-1.medium.com/max/800/0*xAADmPJN52Yy6XJV.jpg"
      isCartMode={true}
      key="1"
      title="WebSockets-MultiChat"
      url="https://github.com/stasrdv/WebSockets-MultiChat-Example"
    />
    <Card
      deleteItem={[Function]}
      description="This The project provides a kick start for angular applications. The project build on top of the angular-cli project."
      id={2}
      img="https://cdn-images-1.medium.com/max/1600/1*juPyda3wq9uz_SNFRLuANg@2x.png"
      isCartMode={true}
      key="2"
      title="Angular4-Boilerplate"
      url="https://github.com/stasrdv/Angular4-seed"
    />
    <Card
      deleteItem={[Function]}
      description="An automatic messaging service that works behind the scenes.TocSin sends a pre-composed message by you to a selected contact."
      id={3}
      img="https://image.winudf.com/v2/image/Y29tLnRhbnhlLmFuZHJvaWQudG9jc2luX2ljb25fMF82MTIyOTM1OA/icon.png?w=170&fakeurl=1&type=.png"
      isCartMode={true}
      key="3"
      title="TocSin-Low-battery-SmS"
      url="https://github.com/stasrdv/TocSin-Low-battery-SmS"
    />
    <Card
      deleteItem={[Function]}
      description="Integrated transport Android application"
      id={5}
      img="https://userscontent2.emaze.com/images/3564aa26-dad4-4a39-b3dd-783f7e9a4953/7c2902b42e42449f05cca53cbd8462eb.png"
      isCartMode={true}
      key="5"
      title="Sapir-On-My-Way"
      url="https://github.com/stasrdv/Sapir-On-My-Way"
    />
    <Card
      deleteItem={[Function]}
      description="Simple Example of book store ,using angular 6 and Google Books API."
      id={6}
      img="https://i.pinimg.com/originals/70/bd/63/70bd630505578fffc4c873001f89a9e1.jpg"
      isCartMode={true}
      key="6"
      title="Angular-6-Book-Store"
      url="https://github.com/stasrdv/Angular-6-Book-Store"
    />
  </div>
</div>
`;

exports[`Cart component should shallow correctly 1`] = `
<div
  className="jumobtron board"
>
  <div
    className="row justify-content-center"
  >
    <Card
      deleteItem={[Function]}
      description="React native todolist implementation"
      id={4}
      img="https://raw.githubusercontent.com/lucasbento/react-native-actions/master/common/media/logo.png"
      isCartMode={true}
      key="4"
      title="React-Native-ToDoList"
      url="https://github.com/stasrdv/React-Native-ToDoList"
    />
    <Card
      deleteItem={[Function]}
      description="WebSockets simple chat example (Socket.io ,ExpressJs,VanillaJS)"
      id={1}
      img="https://cdn-images-1.medium.com/max/800/0*xAADmPJN52Yy6XJV.jpg"
      isCartMode={true}
      key="1"
      title="WebSockets-MultiChat"
      url="https://github.com/stasrdv/WebSockets-MultiChat-Example"
    />
    <Card
      deleteItem={[Function]}
      description="This The project provides a kick start for angular applications. The project build on top of the angular-cli project."
      id={2}
      img="https://cdn-images-1.medium.com/max/1600/1*juPyda3wq9uz_SNFRLuANg@2x.png"
      isCartMode={true}
      key="2"
      title="Angular4-Boilerplate"
      url="https://github.com/stasrdv/Angular4-seed"
    />
    <Card
      deleteItem={[Function]}
      description="An automatic messaging service that works behind the scenes.TocSin sends a pre-composed message by you to a selected contact."
      id={3}
      img="https://image.winudf.com/v2/image/Y29tLnRhbnhlLmFuZHJvaWQudG9jc2luX2ljb25fMF82MTIyOTM1OA/icon.png?w=170&fakeurl=1&type=.png"
      isCartMode={true}
      key="3"
      title="TocSin-Low-battery-SmS"
      url="https://github.com/stasrdv/TocSin-Low-battery-SmS"
    />
    <Card
      deleteItem={[Function]}
      description="Integrated transport Android application"
      id={5}
      img="https://userscontent2.emaze.com/images/3564aa26-dad4-4a39-b3dd-783f7e9a4953/7c2902b42e42449f05cca53cbd8462eb.png"
      isCartMode={true}
      key="5"
      title="Sapir-On-My-Way"
      url="https://github.com/stasrdv/Sapir-On-My-Way"
    />
    <Card
      deleteItem={[Function]}
      description="Simple Example of book store ,using angular 6 and Google Books API."
      id={6}
      img="https://i.pinimg.com/originals/70/bd/63/70bd630505578fffc4c873001f89a9e1.jpg"
      isCartMode={true}
      key="6"
      title="Angular-6-Book-Store"
      url="https://github.com/stasrdv/Angular-6-Book-Store"
    />
  </div>
</div>
`;
